packages/community-cli-plugin/package.json

```diff
@@ -6,7 +6,7 @@
     "react-native",
     "tools"
   ],
-  "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/buid-scripts#readme",
+  "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/community-cli-plugin/README.md",
```

Comment:
Thanks for spotting this! Can we keep the `#readme` anchor link — this navigates to the package folder on GitHub.

```suggestion
  "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/community-cli-plugin#readme",
```

Reply:
@huntie thank you for your suggestion :). I missed it.